lang/octave/README.md

```diff
@@ -0,0 +1,271 @@
+# GNU Octave
+
+![octave-logo](octave-logo.svg)
+
+[GNU Octave](https://www.gnu.org/software/octave/) is a free and open source
+(FOSS) high-level language for array manipulation and linear algebra (very 
+similar to [MATLAB](https://www.mathworks.com/products/matlab.html)). It ships 
+with an IDE and is very good for things like curve-fitting, modeling and 
+parameter optimization, signal processing, and image manipulation.  
+
+In this example, we'll create an Ubuntu 18.04 based container with a basic
+Octave installation. The definition file will allow for easily starting the 
+Octave IDE, or running a script.
+
+## writing the definition file
```

Comment:
Change "writing the definition file" to "Writing the Definition File" to keep the title case compliant with the Chicago Manual of Style.